lib/AuditoManager.dart

```diff
@@ -1,61 +1,77 @@
 import 'package:audioplayers/audioplayers.dart';
 import 'package:flutter/foundation.dart';  // For debugPrint
+import 'storage.dart';  // Assumed to handle all database interactions
 
 class MindfulnessAudioManager {
   final AudioPlayer _audioPlayer = AudioPlayer();
-  bool isPlaying = false;
```

Comment:
You either need an accessor or need to make `_audioPlayer` public. Users can't get the audio player's status as is now.